project/Dependencies.scala

```diff
@@ -180,7 +182,7 @@ object Dependencies {
   val dspApiMainLibraryDependencies = Seq(
     zio,
     zioMacros,
-    zioHttp,
+    zioHttpOld, // TODO: switch
```

Comment:
All 3 repetition of `// TODO: switch` could be removed.

Reply:
done